code/model/editableformfields/EditableDropdown.php

```diff
@@ -1,32 +1,66 @@
 <?php
+
 /**
  * EditableDropdown
  *
  * Represents a modifiable dropdown (select) box on a form
  *
+ * @property bool $UseEmptyString
+ * @property string $EmptyString
+ *
  * @package userforms
  */
-
 class EditableDropdown extends EditableMultipleOptionField
 {
 
     private static $singular_name = 'Dropdown Field';
 
     private static $plural_name = 'Dropdowns';
 
+    private static $db = array(
+        'UseEmptyString' => 'Boolean',
+        'EmptyString' => 'Varchar(255)',
+    );
+
     /**
      * @return FieldList
      */
     public function getCMSFields()
     {
         $fields = parent::getCMSFields();
 
+        $fields->addFieldToTab(
+            'Root.Main',
+            CheckboxField::create('UseEmptyString')
+                ->setTitle('Set default empty string')
+        );
+
+        $fields->addFieldToTab(
+            'Root.Main',
+            TextField::create('EmptyString')
+                ->setTitle('Empty String')
+        );
+
         $fields->removeByName('Default');
 
         return $fields;
     }
 
     /**
+     * @return ValidationResult
+     */
+    public function validate()
+    {
+        $result = parent::validate();
+
+        if ($this->UseEmptyString && !$this->EmptyString) {
+            $result->error('Please enter the default option text');
```

Comment:
I had thought of that and figured I'd see if anyone else brought it up. Skip validation and just set empty quotes if there is no value for that field.